README.md

```diff
@@ -47,7 +47,7 @@ git clone -b master https://github.com/fs/osx-bootstrap.git
 * Node.js stable for writing JavaScript general-purpose code
 * Bundler gem for managing Ruby libraries
 * Tmuxinator gem for managing tmux sessions easily
-* Software installed with Brew Cask in to `/Applications` : iTerm2, HipChat, Skype, Google Chrome,
+* Software installed with Brew Cask in to `/Applications` : iTerm2, Slack, Skype, Google Chrome,
   Firefox, Google Drive, f-lux, Sublime Text 3, Java, Dropbox, The Unarchiver, Dash
```

Comment:
@timurvafin Looks like there are no `f-lux` installation. Should I remove it from `README`?